charts/posthog/README.md

```diff
@@ -16,47 +16,129 @@ This chart bootstraps a [PostHog](https://posthog.com/) deployment on a [Kuberne
 
 ## Prerequisites
 
-- [Google Cloud Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine/)
 - [Kubernetes](https://kubernetes.io/) 1.6+ with Beta APIs enabled
 - [Helm](https://helm.sh/) >= v3
 
+## Setting up Kuberentes Cluster
+
+<details>
+  <summary>
+    <b>Google Cloud</b>
+  </summary>
+<br />
+See [Google Cloud Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine/)
+</details>
+
+<details>
+  <summary>
+    <b>AWS</b>
+  </summary>
+<br />
+TODO
+</details>
+
+<details>
+  <summary>
+    <b>Digital Ocean</b>
+  </summary>
+<br />
+Note that there is a [one click](todo: link) option to deploy Posthog on DigitalOcean
+</details>
+
 ## Installing the chart
 
-To install the chart with the release name `posthog`, run the following:
+To install the chart with the release name `posthog` in `posthog` namespace, run the following:
 
 ```console
 helm repo add posthog https://posthog.github.io/charts-clickhouse/
 helm repo update
-helm install -f values.yaml --timeout 20m posthog posthog/posthog
+helm install -f values.yaml --timeout 20m --create-namespace --namespace posthog posthog posthog/posthog
```

Comment:
Why install in a namespace by default in the instructions here.

I think we can assume the users who want separate namespaces will know how to do so. Encouraging to do so by default:
- increases command complexity, the amount of flags you need to specify when installing
- It makes debugging _anything_ on the cluster more complicated.

If you're installing on a shared cluster, sure that makes sense. However I don't think we should encourage it here.

Reply:
I think if we always use a namespace it works the same way for everyone being on a shared cluster or not. It's also much easier for us to say just nuke the whole namespace with everything in it without needing to verify they are only running Posthog there first. There's an easy command for kubectl to be in a specific namespace, so you don't need to provide this argument every time. That said I don't feel strongly about this, just figured it could make our lives easier in the long run.